cmd/juju/firewall/listformatter.go

```diff
@@ -11,11 +11,12 @@ import (
 	"github.com/juju/errors"
 
 	"github.com/juju/juju/cmd/output"
+	"github.com/juju/juju/core/network/firewall"
 )
 
 type firewallRule struct {
-	KnownService   string   `yaml:"known-service" json:"known-service"`
-	WhitelistCIDRS []string `yaml:"whitelist-subnets,omitempty" json:"whitelist-subnets,omitempty"`
+	KnownService firewall.WellKnownServiceType `yaml:"known-service" json:"known-service"`
+	AllowedCIDRS []string                      `yaml:"allowed-subnets,omitempty" json:"allowed-subnets,omitempty"`
```

Comment:
Are we allowed to change command output without a deprecation warning in the output? We should probably keep the WhitelistCIDRS for another release then kill off with fire.

Reply:
You'll have to fight with Joe over this one

Reply:
I can't see why we can't keep it as it is

Reply:
@manadart I think you've been overruled here

Reply:
Yes sure, we just need QA steps for model migrations between older 2.9/3.0/3.1 controllers.